packages/core/src/components/modal/readme.md

```diff
@@ -0,0 +1,41 @@
+# fl-modal
+
+This is a modal.
+
+<fl-modal header="This is a modal." action="a" actionText="Press Me!" secondary="b" secondaryText="Press Me Too!">
```

Comment:
suggestion: Rename `action` to `control`. The button acts as a `control` to open the modal.